15.-Project-3-Elections-Scraper.py

```diff
@@ -0,0 +1,155 @@
+
+import requests
+from bs4 import BeautifulSoup as BS
+import csv
+import sys
+import os
+import unicodedata
+
+URL = 'https://volby.cz/pls/ps2017nss/ps3?xjazyk=CZ'
+payload = {'area':None,
+           'municipality': None}
+
+
+def parse_command_line():
+    while True:
+        if len(sys.argv) != 3:
+            print('Not correct number of arguments (2)')
+            break
+        else:
+            arg = sys.argv[1:]
+            count = 0
+            for k, v in payload.items():
+                payload[k] = arg[count]
+                count += 1
+            return payload
+
+
+def scrape_area():
+    parsed_command_line = parse_command_line()
+    r = requests.get(URL)
+    soup = BS(r.text, 'html.parser')
+    result_town = None
+    for table in soup.find_all('table'):
+        for child in filter(lambda x:x != '\n', table.children):
+            for info in filter(lambda x:x != '\n', child):
+                if info.string == parsed_command_line['area']:
+                    result_town = child
```

Comment:
You can use `find` keyword argument to find a tag with `string`, no need to do it manually https://www.crummy.com/software/BeautifulSoup/bs4/doc/#the-string-argument